pkg/sdkserver/sdkserver_test.go

```diff
@@ -745,6 +753,14 @@ func TestSDKServerWatchGameServerFeatureSDKWatchSendOnExecute(t *testing.T) {
 
 func TestSDKServerSendGameServerUpdate(t *testing.T) {
 	t.Parallel()
+
+	agruntime.FeatureTestMutex.Lock()
+	defer agruntime.FeatureTestMutex.Unlock()
+	err := agruntime.ParseFeatures(string(agruntime.FeatureSDKWatchSendOnExecute) + "=false")
+	if !assert.NoError(t, err) {
```

Comment:
We should probably stick to:
```
	if err != nil {
		assert.FailNow(t, err.Error())
	}
```
Pattern, as per #1659 

Or if we want to bring in the `require` package, happy to see that as well.